packages/react-composites/src/composites/common/styles/ParticipantContainer.styles.ts

```diff
@@ -94,12 +94,14 @@ export const peoplePaneContainerTokens: IStackTokens = {
 /**
  * @private
  */
-export const participantListWrapper = mergeStyles({
-  width: '20rem',
-  // max width at 50% of view so the People Pane is not squeezing the Message Pane to almost nothing when on small screen or high zoom in
-  maxWidth: '50vw',
-  height: '100%'
-});
+export const participantListWrapper = (theme: Theme): string =>
+  mergeStyles({
+    width: '20rem',
+    // max width at 50% of view so the People Pane is not squeezing the Message Pane to almost nothing when on small screen or high zoom in
+    maxWidth: '50vw',
```

Comment:
Unfortunately we can never use `vw` for styling in our composites, vw is the viewport width, however Contoso doesn't have to make our composite span the entire width of the viewport. If, for example, the composite was sized to smaller of the viewport vh/vw css won't apply correctly